contracts/contracts/strategies/NativeStaking/FeeAccumulator.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+
+import { Governable } from "../../governance/Governable.sol";
+
+/**
+ * @title Fee Accumulator for Native Staking SSV Strategy
+ * @notice Receives execution rewards which includes tx fees and
+ * MEV rewards like tx priority and tx ordering.
+ * It does NOT include swept ETH from beacon chain consensus rewards or full validator withdrawals.
+ * @author Origin Protocol Inc
+ */
+contract FeeAccumulator is Governable {
```

Comment:
I'm wondering if this contract should have a transferToken method that's only callable by the governor and does a safeTransfer of the coin.

Otherwise, we don't use Governable, and could remove it from this contract.

Reply:
I've got rid of `Governable`.

The contract is proxied so if a large amount of ERC20 tokens were accidentally transferred to this contract we could upgrade and recover the tokens. Otherwise, `Governable` doesn't do anything.